containers/airflow/dags/brasil/sinan.py

```diff
@@ -53,25 +55,43 @@ def extract_data(disease):
 
         extract.download(disease)
 
-        logger.info(f"Data for {disease} downloaded at /tmp/pysus")
+        logger.info(f"Data for {disease} extracted")
 
 
     @task(task_id='upload')
     def upload_data():
         """
-        Creates table and upsert all data found in `/tmp/pysus/*.parquet`, 
-        cleaning after inserting
+        This task will upload every disease in the directory /tmp/pysus
+        that ends with `.parquet`, creating the corresponding table
+        to each disease in `brasil` schema. 
         """        
-
-        loading.upload()
+        try:
+            loading.upload()
+        except Exception as e:
+            logger.error(e)
+            raise e
+
+    @task(trigger_rule='all_done')
+    def remove_data_dir():
+        """ 
+        Cleans /tmp/pysus data directory.
+        """
+        shutil.rmtree(PYSUS_DATA_PATH, ignore_errors=True)
+        logger.warning(f'{PYSUS_DATA_PATH} removed')
 
 
+    # `expand` will create a task for each disease/year.
+    # There will be about 35 diseases total, each disease can
+    # contain several years to fetch, setting `max_active_tasks`
+    # to 2 will allow only two diseases to be downloading at 
+    # the same time
     download = extract_data.expand(disease=list(diseases.keys()))
     
     upload = upload_data()
 
+    clean = remove_data_dir()
 
-    start >> download >> upload >> done 
+    start >> download >> upload >> done >> clean
```

Comment:
@luabida I think we should have some upload verification step before done.

Reply:
I've added a step that calculates how many rows were insert into the tables, now apart from throwing error if upload fail, it will store how many new rows were insert into the tables.

Reply:
Good. It will greatly facilitate debugging the DAG